.scalafmt.conf

```diff
@@ -1,3 +1,5 @@
+version = "1.5.1"
```

Comment:
is this generated with codacy-sbt ?

Reply:
No. How can I generate it? I just added the version there.

Reply:
Generated with `genScalafmtConfig`.